remote-api/src/main/java/org/alfresco/rest/api/impl/TagsImpl.java

```diff
@@ -244,8 +246,11 @@ public Tag getTag(StoreRef storeRef, String tagId)
 
     public CollectionWithPagingInfo<Tag> getTags(String nodeId, Parameters params)
     {
-		NodeRef nodeRef = validateTag(nodeId);
-
+		NodeRef nodeRef = nodes.validateNode(nodeId);
+		if( nodeRef == null )
+		{
+			throw new EntityNotFoundException(nodeId);
```

Comment:
Doesn't `nodes.validateNode` already throw this?

Reply:
Reverted the above changes.